include/z64item.h

```diff
@@ -257,16 +257,16 @@ typedef enum {
     /* 0x6D */ ITEM_GORON_RUBY,
     /* 0x6E */ ITEM_ZORA_SAPPHIRE,
     /* 0x6F */ ITEM_STONE_OF_AGONY,
-    /* 0x70 */ ITEM_GERUDO_CARD,
+    /* 0x70 */ ITEM_GERUDOS_CARD,
     /* 0x71 */ ITEM_SKULL_TOKEN,
     /* 0x72 */ ITEM_HEART_CONTAINER,
     /* 0x73 */ ITEM_HEART_PIECE,
-    /* 0x74 */ ITEM_KEY_BOSS,
-    /* 0x75 */ ITEM_COMPASS,
+    /* 0x74 */ ITEM_DUNGEON_BOSS_KEY,
+    /* 0x75 */ ITEM_DUNGEON_COMPASS,
```

Comment:
Don't feel it necessary to prefix these with DUNGEON. The map item is actually called Dungeon Map

Reply:
The prefix is meant to highlight that the three dungeon items boss key/compass/map are in a "group", for example in `Item_Give`:
https://github.com/zeldaret/oot/blob/6b77ddba5ae0941c10c58260c44270dfcb88e182/src/code/z_parameter.c#L1426-L1428
so the group of "dungeon items"

at least that was the intention when I opened #1263 but I've since realized the item enum is a lot more sketchy than that so I'm not so big on this anymore. If someone else also thinks this is better without `dungeon_` I'm fine removing it

But I also think it's okay to have "dungeon"? (not as terrible as `ITEM_SPIRITUAL_KOKIRI_EMERALD` at least)

Reply:
Bump: Is this considered resolved?
I'll assume yes in like 7 days or so